packages/ra-ui-materialui/src/input/SelectInput.tsx

```diff
@@ -115,6 +115,7 @@ export const SelectInput = (props: SelectInputProps) => {
         label,
         loaded,
         loading,
+        margin = 'dense',
```

Comment:
Why a default here and not in the others ? Besides, I think the Form components already have this default.

Reply:
I copy the default from the other inputs.
If you want I can remove all the defaults for margin?

Reply:
But I only see margin's default value only being set for most Inputs like that

Reply:
I forgot we did this, nvm :)